### PR TITLE
Run tests on 4.19 kernel

### PR DESCRIPTION
**Pull request by nebril**

### CODE REVIEW

**Comment by nebril** [comment]
test-with-kernel

**Comment by nebril** [comment]
test-with-kernel

**Comment by nebril** [comment]
test-with-kernel

**Comment by nebril** [comment]
test-with-kernel

File: vagrant_box_defaults.rb [hunk 1]
```diff
@@ -6,3 +6,5 @@
 $SERVER_VERSION= "167"
 $NETNEXT_SERVER_BOX= "cilium/ubuntu-next"
 $NETNEXT_SERVER_VERSION= "45"
+@v419_SERVER_BOX= "cilium/ubuntu-4-19"
```
Comment (errordeveloper) [inline vagrant_box_defaults.rb:9]: Just out of curiosity, where is this one defined? Was there another PR that introduced the image?

Comment (nebril) [reply vagrant_box_defaults.rb:9]: This file is the source of truth about which box/version combo should be used for given branch. This image has been introduced by this PR: https://github.com/cilium/packer-ci-build/pull/197 (and two subsequent fixes)

**Comment by nebril** [comment]
test-with-kernel

**Comment by nebril** [comment]
test-with-kernel

File: vagrant_box_defaults.rb (hunk 1 above)
Comment (aanm) [inline vagrant_box_defaults.rb:9]: What's the difference between the initialization of a variable with `$` and `@`?

File: jenkinsfiles/ginkgo-kernel.Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,166 @@
+@Library('cilium') _
```
Comment (aanm) [inline jenkinsfiles/ginkgo-kernel.Jenkinsfile:1]: Perhaps the filename should be `ginkgo-kernel-4-19`

Comment (nebril) [reply jenkinsfiles/ginkgo-kernel.Jenkinsfile:1]: I just updated the jenkinsfile, this file is parametrized, it runs 4.19 by default, but can run any other future custom kernel versions that we support

File: vagrant_box_defaults.rb (hunk 1 above)
Comment (nebril) [reply vagrant_box_defaults.rb:9]: `@` allows to get variable value by string value with `instance_variable_get`. Variables that don't start with `@` are not allowed to be retrieved like that #justrubythings

**Comment by nebril** [comment]
test-with-kernel

**Comment by nebril** [comment]
test-me-please

**Comment by nebril** [comment]
test-with-kernel --version=419 --focus=K8sDemos.*

**Comment by nebril** [comment]
test-with-kernel --version=419 --focus=K8sDemos.\*

**Comment by nebril** [comment]
test-with-kernel --version=419 --focus=K8sDemos.*

File: vagrant_box_defaults.rb (hunk 1 above)
Comment (errordeveloper) [reply vagrant_box_defaults.rb:9]: Prefixing a variable with actually makes it [global](https://ruby-doc.org/docs/ruby-doc-bundle/UsersGuide/rg/globalvars.html). I don't think there is a reason for using globals in Vagrantfiles, it just seems to have become somewhat of a tradition, probably due to some funny historical detail...

Comment (errordeveloper) [reply vagrant_box_defaults.rb:9]: I mean, people usually copy a Vagrantfile example, and I think there are lots of examples with global vars. Here is one I used hack on: https://github.com/errordeveloper/weave-demos/blob/master/coreos-vagrant/Vagrantfile.

**Comment by nebril** [comment]
test-me-please

**Comment by nebril** [comment]
test-with-kernel --version=419 --focus=K8sDemos.*

**Comment by nebril** [comment]
test-with-kernel --version=419 --focus=K8sDemos.*

**Comment by nebril** [comment]
test-me-please

**Comment by nebril** [comment]
test-me-please

**Pull request merged**